### PR TITLE
Device Service review v2

### PR DESCRIPTION
**Pull request by iain-anderson**
Review of device-uart which is proposed to be adopted into edgexfoundry.
Process and criteria are set out at https://wiki.edgexfoundry.org/display/FA/Process+for+new+Device+Services

Note that if adopted we will be targeting a release based on Levski (EdgeX v2.3)
Previous review discussions here: #3

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+ARG ALPINE_PKG_BASE="make git openssh-client gcc libc-dev zeromq-dev libsodium-dev"
+ARG ALPINE_PKG_EXTRA=""
+
+# set the working directory
+WORKDIR /device-uart
+
+# Replicate the APK repository override.
+# If it is no longer necessary to avoid the CDN mirros we should consider dropping this as it is brittle.
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
```
Comment (cloudxxx8) [inline Dockerfile:28]: These three lines should also be removed

**Review by cloudxxx8 — COMMENTED** [review]
this PR looks good to me for v2.2.x release
do we need to upgrade to v2.3?

**Comment by lenny-intel** [comment]
> do we need to upgrade to v2.3?

Yes, the target for initial release is 2.3.0

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Comment by iain-anderson** [comment]
Service accepted into EdgeX as per TSC vote 3rd May 2023

**Pull request closed without merging**